### PR TITLE
Addl anguages to the missions

### PR DESCRIPTION
**Pull request by privat**
So UI can use it to feed highlighter and the form hidden fields

### CODE REVIEW

File: src/model/loader.nit [hunk 1]
```diff
@@ -58,6 +58,15 @@ redef class AppConfig
 		var html = proc.process(content).write_to_string
 
 		var track = new Track(title_id, title, html)
+
+		var ls = ini["languages"]
+		if ls != null then
```
Comment (lbajolet) [inline src/model/loader.nit:63]: What should we do if the `languages` setting of a track is unavailable? Shoot an error or consider all languages are accepted ?

**Comment by lbajolet** [comment]
+1 aside from that

File: src/model/loader.nit (hunk 1 above)
Comment (privat) [reply src/model/loader.nit:63]: I don't know, this is policy, not mechanism. :)

Comment (Morriar) [reply src/model/loader.nit:63]: I like the "all languages are accepted" but the frontend need to know the list of all these languages. The API should reflect it.

@privat, once you have decided of a policy, you should open an issue about that.

File: tracks/nit/nitcc/config.ini [hunk 2]
```diff
@@ -1,2 +1,3 @@
 title=Elvish Vectorial Images and nitcc
 req=visitor
+languages=plain
```
Comment (Morriar) [inline tracks/nit/nitcc/config.ini:3]: Plain?

Comment (Morriar) [inline tracks/nit/nitcc/config.ini:3]: Maybe this can be the default value for a empty language array?

Comment (privat) [reply tracks/nit/nitcc/config.ini:3]: `plain` is mainly here to test that a mission ini can override the track ini.

However, in this specific mission (from CSG16), the answer was a plain flag (not code).
So, maybe for some real mission we want a direct answer (not code). I used `plain` to explain the UI that we do not want code highlighting.

For the default, I'm not sure what we want: plain? all languages?
Maybe just refuse the load tracks that does not set the languages?

Comment (Morriar) [reply tracks/nit/nitcc/config.ini:3]: > Maybe just refuse the load tracks that does not set the languages?

I think this is the best behavior to adopt for now. Can you make the change in this PR?

Comment (privat) [reply tracks/nit/nitcc/config.ini:3]: OK. new commit pushed

File: src/model/loader.nit (hunk 1 above)
Comment (privat) [reply src/model/loader.nit:63]: mandatory languages solve the issue :)

**Comment by Morriar** [comment]
We were talking about refusing, this is just a warning?+1

**Comment by privat** [comment]
@Morriar commit says "I do not crash or return to allow easier debugging of not-yet-complete tracks."

**Pull request merged**